crates/containerd-shim-wasm/Cargo.toml

```diff
@@ -16,13 +16,16 @@ anyhow = { workspace = true }
 chrono = { workspace = true }
 git-version = "0.3.5"
 containerd-shim = { workspace = true }
+containerd-shim-wasm-test-modules = { workspace = true, optional = true }
 crossbeam = { workspace = true }
+env_logger = { workspace = true, optional = true }
```

Comment:
since these are under dev-dependencies are they needed here?

Reply:
For consumers of the `testing` feature of the library we need to specify it as a normal dependency as well.
All 3 dev-dependencies are also specified as optional normal dependencies.
I'm open to suggestions :smile: